packages/p2p/src/components/orders/order-table/order-table-content.jsx

```diff
@@ -45,16 +46,32 @@ const OrderTableContent = observer(() => {
 
         if (modified_list.length) {
             return (
-                <OrderTableHeader>
-                    <InfiniteDataList
-                        data_list_className='orders__data-list'
-                        has_more_items_to_load={order_store.has_more_items_to_load}
-                        items={modified_list}
-                        keyMapperFn={item => item.id}
-                        loadMoreRowsFn={order_store.loadMoreOrders}
-                        rowRenderer={row_props => <OrderRow {...row_props} />}
-                    />
-                </OrderTableHeader>
+                <React.Fragment>
+                    {isMobile() ? (
+                        <Div100vhContainer height_offset='21rem'>
+                            <div className='orders__separator' />
+                            <InfiniteDataList
+                                data_list_className='orders__data-list'
+                                has_more_items_to_load={order_store.has_more_items_to_load}
+                                items={modified_list}
+                                keyMapperFn={item => item.id}
+                                loadMoreRowsFn={order_store.loadMoreOrders}
+                                rowRenderer={row_props => <OrderRow {...row_props} />}
+                            />
```

Comment:
Could be set to a variable and used in both mobile and desktop versions.

Reply:
Thanks Aaron,
I've created a wrapper component and wrapped the InfiniteDateList and check the mobile and desktop version inside the wrapper